dags/sparsity_diffusion_devx/configs/common.py

```diff
@@ -18,6 +18,8 @@
 
 
 UPGRADE_PIP = "pip install --upgrade pip"
+UPGRADE_SETUPTOOLS = "python -m pip install --upgrade setuptools"
+UPGRADE_PACKAGING = "python -m pip install --upgrade packaging"
```

Comment:
Nit: we could wrap those upgrades together, if it gets longer in future.

Reply:
@RissyRan Sounds good. So we're leaving this as is for now?